tailwind.config.js

```diff
@@ -2,4 +2,14 @@ module.exports = {
   plugins: [
     require('@tailwindcss/ui')
   ],
+  theme: {
+    extend: {
+      colors: {
+        'zinc-purple': '#332658',
+        'zinc-dark-purple': '#1F1735',
+        'zinc-hot-pink': '#F43EF3',
+        'zinc-pastel-pink': '#FE9BFF',
```

Comment:
I like that we're starting to define our custom pallette; and I'm trying to think of the "appropriate" way to bring forward the way the colors fit into our pallete.

Purple/Pink are [analogous colors](https://www.shutterstock.com/blog/complete-guide-color-in-design#analogous-colors); which means we're likely going to be playing with tints, tones, saturation and shades to create contrast; so names that give some indication of the color-strength may be worthwhile (this, I think, is why Tailwind uses purple-50/purple-500,etc)

Tailwind [suggests using "primary/secondary" and "accent"](https://tailwindcss.com/docs/customizing-colors#naming-your-colors) when naming colors; which I think makes some sense as well for what we're doing; but for now I think this is good enough!

Reply:
Yeah, I was wondering if we should call the colors something like "main" and "secondary", and make it easier to change the entire palette later, but I wasn't sure how "set" the palette is. It felt right to go with something very explicit for now, which we can change later, as things solidify.

Another option that's available with Tailwind is that we can override the "purple" and "pink" colors, including adding dark/light/default variations for those. I was hesitant to override Tailwind defaults simply to reduce the potential for surprise (if you didn't know that "purple" was ovewritten in the main config, you'd try to use it and be surprised it doesn't match what the Tailwind docs say it should be).

Reply:
Hmmm, I was thinking we'd go the opposite direction; and embrace the TailwindCSS colors for purple/pink since they've done the work to balance their tones/tints/saturation for contrast.

So `primary` may be purple-900, with `background` being pink-50 and `accent` being `pink-800` (or whatever is actually appropriate)

Reply:
What's up on https://www.zinc.coop/ right now is using the default `purple-900` for its background, and I personally find it a little bit too intensely bright when it's taking up the entire window. This `purple-900` is the darkest color they have in their purple palette (https://tailwindcss.com/docs/customizing-colors). In that respect, I like the darker purples in the mocks better -- they feel a little more soothing and pleasant to me.

Reply:
Ahhh, so maybe what this is is purple-1000! I agree the darker purple is nicer :D